### PR TITLE
RHDEVDOCS-6068 - Updated pre-reqs

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.13

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6068

Link to docs preview:

Pre-reqs for Configuring SSO for Argo CD using Keycloak: https://77143--ocpdocs-pr.netlify.app/openshift-gitops/latest/accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.html

SME review: @chetan-rns 
QE review: @varshab1210
Internal peer review: @eromanova97
Peer review:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc [hunk 1]
```diff
@@ -13,6 +13,7 @@ After the {gitops-title} Operator is installed, Argo CD automatically creates a
 * Red Hat SSO is installed on the cluster.
 * The {gitops-title} Operator is installed on your {OCP} cluster.
 * Argo CD is installed on the cluster.
+* The `DeploymentConfig` API is available in the cluster. For more information, see "DeploymentConfig [apps.openshift.io/v1".
```
Comment (Srivaralakshmi) [inline accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc:16]:
```suggestion
* The `DeploymentConfig` API is available in the cluster. For more information, see "DeploymentConfig [apps.openshift.io/v1]".
```

File: accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc [hunk 2]
```diff
@@ -29,3 +30,8 @@ include::modules/gitops-uninstall-keycloak.adoc[leveloffset=+1]
 * link:https://stedolan.github.io/jq/[`jq` command-line JSON processor documentation.]
 * link:https://argoproj.github.io/argo-cd/operator-manual/rbac/[Argo CD upstream documentation, RBAC Configuration section].
 ////
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.openshift.com/container-platform/latest/rest_api/workloads_apis/deploymentconfig-apps-openshift-io-v1.html[DeploymentConfig [apps.openshift.io/v1]
```
Comment (Srivaralakshmi) [inline accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc:37]:
```suggestion
* link:https://docs.openshift.com/container-platform/latest/rest_api/workloads_apis/deploymentconfig-apps-openshift-io-v1.html[DeploymentConfig [apps.openshift.io/v1]]
```

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 PTAL at my suggestions. Thanks for the good work!

File: accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc:16]: Fixed.

**Comment by chetan-rns** [comment]
Looks good @Dhruv-Soni11 

/lgtm

**Comment by varshab1210** [comment]
LGTM, thanks

File: accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc [hunk 3]
```diff
@@ -13,6 +13,7 @@ After the {gitops-title} Operator is installed, Argo CD automatically creates a
 * Red Hat SSO is installed on the cluster.
 * The {gitops-title} Operator is installed on your {OCP} cluster.
 * Argo CD is installed on the cluster.
+* The `DeploymentConfig` API is available in the cluster. For more information, see "DeploymentConfig [apps.openshift.io/v1]".
```
Comment (varshab1210) [inline accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc:16]: Do we want to link the doc from here?
https://docs.openshift.com/container-platform/latest/rest_api/workloads_apis/deploymentconfig-apps-openshift-io-v1.html

Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc:16]: Hello @varshab1210 - As per our doc standards, we do not add links in the pre-reqs section and list it separately under the Additional resources section. I have added that link in that section for reference purposes.

File: accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc [hunk 4]
```diff
@@ -29,3 +30,8 @@ include::modules/gitops-uninstall-keycloak.adoc[leveloffset=+1]
 * link:https://stedolan.github.io/jq/[`jq` command-line JSON processor documentation.]
 * link:https://argoproj.github.io/argo-cd/operator-manual/rbac/[Argo CD upstream documentation, RBAC Configuration section].
 ////
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.openshift.com/container-platform/latest/rest_api/workloads_apis/deploymentconfig-apps-openshift-io-v1.html[DeploymentConfig [apps.openshift.io/v1]]
```
Comment (bergerhoffer) [inline accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc:37]: Just a suggestion that you could consider also linking to the docs on the capability that is what you enable/disable to get the deploymentconfig API: https://docs.openshift.com/container-platform/4.14/installing/cluster-capabilities.html#deployment-config-capability_cluster-capabilities

**Review by bergerhoffer — COMMENTED** [review]
LGTM, just a suggestion of another place you could consider linking to!

File: accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc (hunk 4 above)
Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc]: Fixed. Could you please take a look at this @bergerhoffer?

**Pull request merged**